conf/messages.en

```diff
@@ -2,6 +2,9 @@ service.name=Internal Exports Movements Service
 header.homepage.alt=Go to the HMRC homepage
 header.homepage.text=HM Revenue & Customs
 
+feedback=This is a new service - your {0} will help us to improve it.
+feedback.link=feedback
```

Comment:
Spaces after and before `=` make message more readable for the developers. It's not a bug, it's more like hint

Reply:
Added. The same should be done for the `external-frontend`.